data/Anshul439.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Anshul Wadhwa",
+  "bio": "Pursuing Bachelor's in Computer Science",
+  "links": [
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "http://www.linkedin.com/in/anshul-wadhwa",
+      "icon": "FaLinkedIn"
+    }
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Anshul_439",
+      "icon": "FaTwitter"
+    }
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Anshul439",
+      "icon": "FaTwitter"
```

Also, add a missing comma here. And this should be a GitHub icon as `FaGithub`

```suggestion
    },
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/Anshul439",
      "icon": "FaGithub"
```